nl/guides/for-students/exercises/README.md

```diff
@@ -0,0 +1,199 @@
+---
+title: Oefeningen handleiding
```

Comment:
Oefeningenhandleiding should be in one word, but I think "Oefeningen oplossen" is a better title?
Similar for the course page.